evap/contributor/templates/contributor_index.html

```diff
@@ -96,6 +96,10 @@
                                 {% else %}
                                     {% if course.avg_grade %}
                                         <div class="grade-bg text-center vertically-aligned" style="background-color: {{ course.avg_grade|gradecolor }}" data-toggle="tooltip" data-placement="top" title="{% trans 'Average grade' %}">{{ course.avg_grade|floatformat:1 }}</div>
+                                    {% else %}
+                                        <div class="grade-bg text-center vertically-aligned grade-bg-disabled" data-toggle="tooltip" data-placement="top" title="{% trans 'Not enough votes to publish results' %}">
```

Comment:
is it intentional that this view does not contain the distribution bar?

Reply:
it was intentional not to include the deviation when we added the grade. but that might have been because we were feeling the deviation doesn't help. we could add the distribution bar here, that would take up more space but would be consistent to the other views. what do you guys think?